### ISSUE TITLE
Insight : Definition object should contain 'type' property: Object({ type: undefined, resolve: [function resolve] })

### ISSUE THREAD

**Issue by GhosTHaise** [directus/directus#22553]
### Describe the Bug

I am trying to build insight with supabase + directus . I have import all tables from supapase as a collection model on directus. When i try add list that include any collections i got this error :
![image](https://github.com/directus/directus/assets/68249758/b6a00d2f-407b-400f-b4dd-a1329b7610a5)

### To Reproduce

Create new directus project and link this to supabase postgres. I have a user table which have relationship to auth.users used to supabase auth . 

### Directus Version

v 10.11.0

### Hosting Strategy

Self-Hosted (Custom)

### Database

PostgreSQL 16.

**Comment by br41nslug** [comment]
> I have a user table which have relationship to auth.users used to supabase auth .

Have you configured this `auth.users` collection and the relationship properly in directus?

**Comment by GhosTHaise** [comment]
my full log : 
```bash
   ╭───────────────────────────────────────────────────╮
   │                                                   │
   │                 Update available!                 │
   │                                                   │
   │                 10.11.0 → 10.11.1                 │
   │                 1 version behind                  │
   │                                                   │
   │                 More information:                 │
   │   https://github.com/directus/directus/releases   │
   │                                                   │
   ╰───────────────────────────────────────────────────╯

error: WITH geometries as (
                                        select * from geometry_columns
                                        union
                                        select * from geography_columns
                                )
                                SELECT f_table_name as table_name
                                        , f_geometry_column as column_name
                                        , type as data_type
                                FROM geometries g
                                JOIN information_schema.tables t
                                        ON g.f_table_name = t.table_name
                                        AND t.table_type = 'BASE TABLE'
                                WHERE f_table_schema in ($1,$2)
                                 - relation "geometry_columns" does not exist
    at Parser.parseErrorMessage (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:283:98)
    at Parser.handlePacket (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:122:29)
    at Parser.parse (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:35:38)
    at Socket.<anonymous> (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/index.js:11:42)
    at Socket.emit (node:events:517:28)
    at addChunk (node:internal/streams/readable:335:12)
    at readableAddChunk (node:internal/streams/readable:308:9)
    at Readable.push (node:internal/streams/readable:245:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 115,
  severity: 'ERROR',
  code: '42P01',
  detail: undefined,
  hint: undefined,
  position: '41',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'parse_relation.c',
  line: '1392',
  routine: 'parserOpenTable'
}
[15:02:46.784] WARN: Couldn't load extensions
[15:02:46.784] WARN: WITH geometries as (
                                        select * from geometry_columns
                                        union
                                        select * from geography_columns
                                )
                                SELECT f_table_name as table_name
                                        , f_geometry_column as column_name
                                        , type as data_type
                                FROM geometries g
                                JOIN information_schema.tables t
                                        ON g.f_table_name = t.table_name
                                        AND t.table_type = 'BASE TABLE'
                                WHERE f_table_schema in ($1,$2)
                                 - relation "geometry_columns" does not exist
    err: {
      "type": "DatabaseError",
      "message": "WITH geometries as (\n\t\t\t\t\tselect * from geometry_columns\n\t\t\t\t\tunion\n\t\t\t\t\tselect * from geography_columns\n\t\t\t\t)\n\t\t\t\tSELECT f_table_name as table_name\n\t\t\t\t\t, f_geometry_column as column_name\n\t\t\t\t\t, type as data_type\n\t\t\t\tFROM geometries g\n\t\t\t\tJOIN information_schema.tables t\n\t\t\t\t\tON g.f_table_name = t.table_name\n\t\t\t\t\tAND t.table_type = 'BASE TABLE'\n\t\t\t\tWHERE f_table_schema in ($1,$2)\n\t\t\t\t - relation \"geometry_columns\" does not exist",
      "stack":
          error: WITH geometries as (
                                                select * from geometry_columns
                                                union
                                                select * from geography_columns
                                        )
                                        SELECT f_table_name as table_name
                                                , f_geometry_column as column_name
                                                , type as data_type
                                        FROM geometries g
                                        JOIN information_schema.tables t
                                                ON g.f_table_name = t.table_name
                                                AND t.table_type = 'BASE TABLE'
                                        WHERE f_table_schema in ($1,$2)
                                         - relation "geometry_columns" does not exist
              at Parser.parseErrorMessage (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:283:98)
              at Parser.handlePacket (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:122:29)
              at Parser.parse (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:35:38)
              at Socket.<anonymous> (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/index.js:11:42)
              at Socket.emit (node:events:517:28)
              at addChunk (node:internal/streams/readable:335:12)
              at readableAddChunk (node:internal/streams/readable:308:9)
              at Readable.push (node:internal/streams/readable:245:10)
              at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
      "length": 115,
      "name": "error",
      "severity": "ERROR",
      "code": "42P01",
      "position": "41",
      "file": "parse_relation.c",
      "line": "1392",
      "routine": "parserOpenTable"
    }
[15:02:47.027] WARN: "PUBLIC_URL" should be a full URL```

**Comment by GhosTHaise** [comment]
> > I have a user table which have relationship to auth.users used to supabase auth .
> 
> Have you configured this `auth.users` collection and the relationship properly in directus? Because the error looks like it is trying to relate to a non-existent table.
> 
> have you tried adding the auth schema to directus search_path? DB_SEARCH_PATH="public,auth"

my full log :

   ╭───────────────────────────────────────────────────╮
   │                                                   │
   │                 Update available!                 │
   │                                                   │
   │                 10.11.0 → 10.11.1                 │
   │                 1 version behind                  │
   │                                                   │
   │                 More information:                 │
   │   https://github.com/directus/directus/releases   │
   │                                                   │
   ╰───────────────────────────────────────────────────╯

error: WITH geometries as (
                                        select * from geometry_columns
                                        union
                                        select * from geography_columns
                                )
                                SELECT f_table_name as table_name
                                        , f_geometry_column as column_name
                                        , type as data_type
                                FROM geometries g
                                JOIN information_schema.tables t
                                        ON g.f_table_name = t.table_name
                                        AND t.table_type = 'BASE TABLE'
                                WHERE f_table_schema in ($1,$2)
                                 - relation "geometry_columns" does not exist
    at Parser.parseErrorMessage (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:283:98)
    at Parser.handlePacket (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:122:29)
    at Parser.parse (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:35:38)
    at Socket.<anonymous> (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/index.js:11:42)
    at Socket.emit (node:events:517:28)
    at addChunk (node:internal/streams/readable:335:12)
    at readableAddChunk (node:internal/streams/readable:308:9)
    at Readable.push (node:internal/streams/readable:245:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 115,
  severity: 'ERROR',
  code: '42P01',
  detail: undefined,
  hint: undefined,
  position: '41',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'parse_relation.c',
  line: '1392',
  routine: 'parserOpenTable'
}
[15:02:46.784] WARN: Couldn't load extensions
[15:02:46.784] WARN: WITH geometries as (
                                        select * from geometry_columns
                                        union
                                        select * from geography_columns
                                )
                                SELECT f_table_name as table_name
                                        , f_geometry_column as column_name
                                        , type as data_type
                                FROM geometries g
                                JOIN information_schema.tables t
                                        ON g.f_table_name = t.table_name
                                        AND t.table_type = 'BASE TABLE'
                                WHERE f_table_schema in ($1,$2)
                                 - relation "geometry_columns" does not exist
    err: {
      "type": "DatabaseError",
      "message": "WITH geometries as (\n\t\t\t\t\tselect * from geometry_columns\n\t\t\t\t\tunion\n\t\t\t\t\tselect * from geography_columns\n\t\t\t\t)\n\t\t\t\tSELECT f_table_name as table_name\n\t\t\t\t\t, f_geometry_column as column_name\n\t\t\t\t\t, type as data_type\n\t\t\t\tFROM geometries g\n\t\t\t\tJOIN information_schema.tables t\n\t\t\t\t\tON g.f_table_name = t.table_name\n\t\t\t\t\tAND t.table_type = 'BASE TABLE'\n\t\t\t\tWHERE f_table_schema in ($1,$2)\n\t\t\t\t - relation \"geometry_columns\" does not exist",
      "stack":
          error: WITH geometries as (
                                                select * from geometry_columns
                                                union
                                                select * from geography_columns
                                        )
                                        SELECT f_table_name as table_name
                                                , f_geometry_column as column_name
                                                , type as data_type
                                        FROM geometries g
                                        JOIN information_schema.tables t
                                                ON g.f_table_name = t.table_name
                                                AND t.table_type = 'BASE TABLE'
                                        WHERE f_table_schema in ($1,$2)
                                         - relation "geometry_columns" does not exist
              at Parser.parseErrorMessage (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:283:98)
              at Parser.handlePacket (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:122:29)
              at Parser.parse (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/parser.js:35:38)
              at Socket.<anonymous> (/home/ghosthaise/dev/tottem-project/node_modules/pg-protocol/dist/index.js:11:42)
              at Socket.emit (node:events:517:28)
              at addChunk (node:internal/streams/readable:335:12)
              at readableAddChunk (node:internal/streams/readable:308:9)
              at Readable.push (node:internal/streams/readable:245:10)
              at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
      "length": 115,
      "name": "error",
      "severity": "ERROR",
      "code": "42P01",
      "position": "41",
      "file": "parse_relation.c",
      "line": "1392",
      "routine": "parserOpenTable"
    }
[15:02:47.027] WARN: "PUBLIC_URL" should be a full URL```

**Comment by rijkvanzanten** [comment]
I'm not able to reproduce this with the reproduction steps provided against the current version of Supabase cloud + Directus v10.13.1 so I'll close this for now. Happy to reopen if more information becomes available 🙂

**Comment by b3nab** [comment]
Right now I have the same problem, deployed with docker and tried version 10.4.3 and 11.0.2 (latest) and I have the same problem.

Just a note: it was running fine with just a connection string before I started messing up with db search path to try to install directus tables on a dedicated schema on supabase. I'm trying to investigate on this since I really need to have directus NOT installed in public schema because that will be strictly for my main app.

**Comment by b3nab** [comment]
So found a solution for everyone who wants to have Directus with Supabase but into a different schema (for example "directus") where store all directus tables without polluting the public schema which is dedicated to the "main" application.

You can configure the DB_SEARCH_PATH as env to Directus and basically you pass a list of schemas where to find tables. Now, for Supabase when you install the postgis extension it is installed inside "extensions" schema, so if you want to use a custom schema or reference to other supabase schemas (like auth) you have to explicitly reference them inside the search path.

So basically, you can add this env variable:

```
DB_SEARCH_PATH=array:directus,public,auth,extensions
```
And you can add whatever schema you want or need. ✌️

**Comment by b3nab** [comment]
Sorry, I was extremely happy to have it functioning for a couple of minutes that I rushed here writing the comment before.
After a while (not so much) it became inaccessible and unusable responding with this generic error: {
  "errors": [
    {
      "message": "An unexpected error occurred.",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}

<img width="622" alt="Screenshot 2024-08-23 alle 06 08 43" src="https://github.com/user-attachments/assets/38a38c08-ae74-43d9-bbc9-132f15824295">


Docker logs are the following (when I try to navigate to the admin):

```
2024-08-23T04:06:21.041452587Z [04:06:20] GET /admin/login 200 1ms
2024-08-23T04:06:21.041697827Z [04:06:20] GET /admin/assets/runtime-core.esm-bundler-zlSgKseS.js 200 5ms
2024-08-23T04:06:21.041706227Z [04:06:20] GET /admin/assets/vue.runtime.esm-bundler-BY4gCD-H.js 200 5ms
2024-08-23T04:06:21.041709427Z [04:06:20] GET /admin/assets/pinia.ZGOsqJmi.entry.js 200 6ms
2024-08-23T04:06:21.041712067Z [04:06:20] GET /admin/assets/vue-i18n.upvyfvU5.entry.js 200 4ms
2024-08-23T04:06:21.041728547Z [04:06:20] GET /admin/assets/vue-router.Dt24BTtJ.entry.js 200 5ms
2024-08-23T04:06:21.041789908Z [04:06:20] GET /admin/assets/index-CHzbp3wQ.css 200 16ms
2024-08-23T04:06:21.041824988Z [04:06:20] GET /admin/assets/index-DqDL7Qf0.js 200 11ms
2024-08-23T04:06:21.041829108Z [04:06:20] GET /admin/assets/index.ZEGekNQg.entry.js 200 34ms
2024-08-23T04:06:22.466563217Z [04:06:22.299] ERROR: select 1 from "directus_sessions" where "token" = $1 and "user" = $2 and "share" is null and "expires" >= $3 limit $4 - relation "directus_sessions" does not exist
2024-08-23T04:06:22.466596577Z     err: {
2024-08-23T04:06:22.466599777Z       "type": "DatabaseError",
2024-08-23T04:06:22.466602417Z       "message": "select 1 from \"directus_sessions\" where \"token\" = $1 and \"user\" = $2 and \"share\" is null and \"expires\" >= $3 limit $4 - relation \"directus_sessions\" does not exist",
2024-08-23T04:06:22.466606537Z       "stack":
2024-08-23T04:06:22.466608978Z           error: select 1 from "directus_sessions" where "token" = $1 and "user" = $2 and "share" is null and "expires" >= $3 limit $4 - relation "directus_sessions" does not exist
2024-08-23T04:06:22.466612258Z               at Parser.parseErrorMessage (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:283:98)
2024-08-23T04:06:22.466615018Z               at Parser.handlePacket (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:122:29)
2024-08-23T04:06:22.466617778Z               at Parser.parse (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:35:38)
2024-08-23T04:06:22.466620498Z               at Socket.<anonymous> (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/index.js:11:42)
2024-08-23T04:06:22.466623738Z               at Socket.emit (node:events:517:28)
2024-08-23T04:06:22.466626618Z               at addChunk (node:internal/streams/readable:368:12)
2024-08-23T04:06:22.466629178Z               at readableAddChunk (node:internal/streams/readable:341:9)
2024-08-23T04:06:22.466631778Z               at Readable.push (node:internal/streams/readable:278:10)
2024-08-23T04:06:22.466634458Z               at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
2024-08-23T04:06:22.466652218Z       "length": 116,
2024-08-23T04:06:22.466654778Z       "name": "error",
2024-08-23T04:06:22.466657058Z       "severity": "ERROR",
2024-08-23T04:06:22.466659218Z       "code": "42P01",
2024-08-23T04:06:22.466661298Z       "position": "15",
2024-08-23T04:06:22.466663458Z       "file": "parse_relation.c",
2024-08-23T04:06:22.466665618Z       "line": "1392",
2024-08-23T04:06:22.466667698Z       "routine": "parserOpenTable"
2024-08-23T04:06:22.466669898Z     }
2024-08-23T04:06:22.979532020Z [04:06:22.789] ERROR: select 1 from "directus_sessions" where "token" = $1 and "user" = $2 and "share" is null and "expires" >= $3 limit $4 - relation "directus_sessions" does not exist
2024-08-23T04:06:22.979562820Z     err: {
2024-08-23T04:06:22.979567260Z       "type": "DatabaseError",
2024-08-23T04:06:22.979570460Z       "message": "select 1 from \"directus_sessions\" where \"token\" = $1 and \"user\" = $2 and \"share\" is null and \"expires\" >= $3 limit $4 - relation \"directus_sessions\" does not exist",
2024-08-23T04:06:22.979574540Z       "stack":
2024-08-23T04:06:22.979577140Z           error: select 1 from "directus_sessions" where "token" = $1 and "user" = $2 and "share" is null and "expires" >= $3 limit $4 - relation "directus_sessions" does not exist
2024-08-23T04:06:22.979580580Z               at Parser.parseErrorMessage (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:283:98)
2024-08-23T04:06:22.979583220Z               at Parser.handlePacket (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:122:29)
2024-08-23T04:06:22.979585700Z               at Parser.parse (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:35:38)
2024-08-23T04:06:22.979588220Z               at Socket.<anonymous> (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/index.js:11:42)
2024-08-23T04:06:22.979590820Z               at Socket.emit (node:events:517:28)
2024-08-23T04:06:22.979593060Z               at addChunk (node:internal/streams/readable:368:12)
2024-08-23T04:06:22.979595300Z               at readableAddChunk (node:internal/streams/readable:341:9)
2024-08-23T04:06:22.979597900Z               at Readable.push (node:internal/streams/readable:278:10)
2024-08-23T04:06:22.979600220Z               at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
2024-08-23T04:06:22.979602540Z       "length": 116,
2024-08-23T04:06:22.979604660Z       "name": "error",
2024-08-23T04:06:22.979606820Z       "severity": "ERROR",
2024-08-23T04:06:22.979609020Z       "code": "42P01",
2024-08-23T04:06:22.979611100Z       "position": "15",
2024-08-23T04:06:22.979613300Z       "file": "parse_relation.c",
2024-08-23T04:06:22.979630581Z       "line": "1392",
2024-08-23T04:06:22.979632981Z       "routine": "parserOpenTable"
2024-08-23T04:06:22.979635101Z     }
2024-08-23T04:06:22.980083222Z [04:06:22.852] ERROR: select "directus_settings"."project_name", "directus_settings"."project_descriptor", "directus_settings"."project_logo", "directus_settings"."project_color", "directus_settings"."default_appearance", "directus_settings"."default_theme_light", "directus_settings"."default_theme_dark", "directus_settings"."theme_light_overrides", "directus_settings"."theme_dark_overrides", "directus_settings"."default_language", "directus_settings"."public_foreground", "directus_settings"."public_favicon", "directus_settings"."public_note", "directus_settings"."custom_css", "directus_settings"."public_registration", "directus_settings"."public_registration_verify_email", "directus_settings"."public_background", "directus_settings"."id" from "directus_settings" order by "directus_settings"."id" asc limit $1 - relation "directus_settings" does not exist
2024-08-23T04:06:22.980094102Z     err: {
2024-08-23T04:06:22.980096382Z       "type": "DatabaseError",
2024-08-23T04:06:22.980098622Z       "message": "select \"directus_settings\".\"project_name\", \"directus_settings\".\"project_descriptor\", \"directus_settings\".\"project_logo\", \"directus_settings\".\"project_color\", \"directus_settings\".\"default_appearance\", \"directus_settings\".\"default_theme_light\", \"directus_settings\".\"default_theme_dark\", \"directus_settings\".\"theme_light_overrides\", \"directus_settings\".\"theme_dark_overrides\", \"directus_settings\".\"default_language\", \"directus_settings\".\"public_foreground\", \"directus_settings\".\"public_favicon\", \"directus_settings\".\"public_note\", \"directus_settings\".\"custom_css\", \"directus_settings\".\"public_registration\", \"directus_settings\".\"public_registration_verify_email\", \"directus_settings\".\"public_background\", \"directus_settings\".\"id\" from \"directus_settings\" order by \"directus_settings\".\"id\" asc limit $1 - relation \"directus_settings\" does not exist",
2024-08-23T04:06:22.980105942Z       "stack":
2024-08-23T04:06:22.980108582Z           error: select "directus_settings"."project_name", "directus_settings"."project_descriptor", "directus_settings"."project_logo", "directus_settings"."project_color", "directus_settings"."default_appearance", "directus_settings"."default_theme_light", "directus_settings"."default_theme_dark", "directus_settings"."theme_light_overrides", "directus_settings"."theme_dark_overrides", "directus_settings"."default_language", "directus_settings"."public_foreground", "directus_settings"."public_favicon", "directus_settings"."public_note", "directus_settings"."custom_css", "directus_settings"."public_registration", "directus_settings"."public_registration_verify_email", "directus_settings"."public_background", "directus_settings"."id" from "directus_settings" order by "directus_settings"."id" asc limit $1 - relation "directus_settings" does not exist
2024-08-23T04:06:22.980114902Z               at Parser.parseErrorMessage (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:283:98)
2024-08-23T04:06:22.980117422Z               at Parser.handlePacket (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:122:29)
2024-08-23T04:06:22.980124502Z               at Parser.parse (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/parser.js:35:38)
2024-08-23T04:06:22.980127222Z               at Socket.<anonymous> (/directus/node_modules/.pnpm/pg-protocol@1.6.1/node_modules/pg-protocol/dist/index.js:11:42)
2024-08-23T04:06:22.980129782Z               at Socket.emit (node:events:517:28)
2024-08-23T04:06:22.980131942Z               at addChunk (node:internal/streams/readable:368:12)
2024-08-23T04:06:22.980134142Z               at readableAddChunk (node:internal/streams/readable:341:9)
2024-08-23T04:06:22.980136382Z               at Readable.push (node:internal/streams/readable:278:10)
2024-08-23T04:06:22.980138582Z               at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
2024-08-23T04:06:22.980140822Z       "length": 117,
2024-08-23T04:06:22.980142902Z       "name": "error",
2024-08-23T04:06:22.980145022Z       "severity": "ERROR",
2024-08-23T04:06:22.980147142Z       "code": "42P01",
2024-08-23T04:06:22.980149262Z       "position": "733",
2024-08-23T04:06:22.980151422Z       "file": "parse_relation.c",
2024-08-23T04:06:22.980155942Z       "line": "1392",
2024-08-23T04:06:22.980158062Z       "routine": "parserOpenTable"
2024-08-23T04:06:22.980160142Z     }
2024-08-23T04:06:23.091831434Z [04:06:22] GET /extensions/sources/index.js 500 18ms
2024-08-23T04:06:23.092395515Z [04:06:22] GET /favicon.ico 404 67ms
2024-08-23T04:06:23.092445715Z [04:06:22] POST /auth/refresh 200 368ms
2024-08-23T04:06:23.092455995Z [04:06:22] GET /auth?sessionOnly 500 18ms
2024-08-23T04:06:23.092489436Z [04:06:22] GET /admin/assets/material-symbols-BLTCXYgc.woff2 200 3ms
2024-08-23T04:06:23.092492076Z [04:06:22] GET /server/info 500 84ms
2024-08-23T04:06:23.092494356Z [04:06:22] GET /admin/assets/Inter-Bold-d8J9BkrN.woff2 200 6ms
2024-08-23T04:06:23.092520276Z [04:06:22] GET /admin/assets/Inter-Medium-IZr8HYrM.woff2 200 6ms
2024-08-23T04:06:23.092523836Z [04:06:22] GET /admin/assets/FiraMono-Medium-CE-6Frqh.woff2 200 6ms
2024-08-23T04:06:23.092558636Z [04:06:22] GET /admin/assets/Inter-SemiBold-1vGiIFm-.woff2 200 4ms
```
